features/step_definitions/hooks.js

```diff
@@ -15,7 +15,7 @@ module.exports = function () {
         this.setOrigin(this.DEFAULT_ORIGIN);
         callback();
     });
-
+/*
```

Comment:
@lbud I had to remove hooks here, since the skip seems to be not supported anymore? Can you have a look and see if I am breaking stuff here?